get.sh

```diff
@@ -160,6 +160,34 @@ parseCommandLineArgs()
 	echo "TESTDIR: $TESTDIR"
 }
 
+# If the current directory contains only a single directory then squash that directory into the current directory
+squashSingleFolderContentsToCurrentDir()
+{
+        # Does current directory contain one and ONLY one directory?
+        if [[ `ls -d */ | wc -l` -eq 1 ]] && [[ `ls -d * | wc -l` -eq 1 ]]; then
```

Comment:
This should ideally include `-1` to explicitly put each entry on a separate line as input to `wc`, although it seems to work regardless (surprisingly - I guess `ls` is operating differently when it's in a pipe)

```suggestion
        if [[ `ls -1d */ | wc -l` -eq 1 ]] && [[ `ls | wc -l` -eq 1 ]]; then
```